relations/src/r1cs/constraint_system.rs

```diff
@@ -340,7 +341,11 @@ impl<F: Field> ConstraintSystem<F> {
                     let lc = transformed_lc_map
                         .get(&lc_index)
                         .expect("should be inlined");
-                    transformed_lc.extend((lc * coeff).0.into_iter());
```

Comment:
Why did this change?

Reply:
In my local attempt it appears that "<&LinearCombination<F> as Mul<F>>::mul" is needed, otherwise the compiler is complaining. Actually, let me try again to see if this is still the case.

Reply:
A smaller change is made, so we continue to use `transformed_lc.extend((lc * coeff).0.into_iter());`

with the only change in signaling to the compiler the following:
```diff
- let mut transformed_lc_map = BTreeMap::new();
+ let mut transformed_lc_map = BTreeMap::<_, LinearCombination<F>>::new();
```

Reply:
Just found out "```diff"

Reply:
An approval will be needed.